parking_permits/cron.py

```diff
@@ -1,29 +1,49 @@
 import logging
 
-from django.utils import timezone
+from dateutil.relativedelta import relativedelta
+from django.utils import timezone as tz
 
 from parking_permits.exceptions import ParkkihubiPermitError
 from parking_permits.models import Customer, ParkingPermit
 from parking_permits.models.parking_permit import ParkingPermitStatus
+from parking_permits.services.mail import PermitEmailType, send_permit_email
 
-logger = logging.getLogger("db")
+logger = logging.getLogger("django")
+db_logger = logging.getLogger("db")
 
 
 def automatic_expiration_of_permits():
     ParkingPermit.objects.filter(
-        end_time__lt=timezone.now(), status=ParkingPermitStatus.VALID
+        end_time__lt=tz.localdate(tz.now()), status=ParkingPermitStatus.VALID
     ).update(status=ParkingPermitStatus.CLOSED)
```

Comment:
I guess we should also send a confirmation email to the user about the permit that has been ended.